python_http_client/client.py

```diff
@@ -62,6 +63,9 @@ def to_dict(self):
 class Client(object):
     """Quickly and easily access any REST or REST-like API."""
 
+    # These are the supported HTTP verbs
+    methods = set(('delete', 'get', 'patch', 'post', 'put'))
```

Comment:
`set` literal instead?

```python
methods = {'delete', 'get', 'patch', 'post', 'put'}
```

Reply:
At time this PR was created, http client claimed support for 2.6

Reply:
@3lnc 

Since that doesn't seem to be the case now... set literal?

Reply:
These are the supported versions going forward: https://github.com/sendgrid/sendgrid-python/blob/v4/.travis.yml#L4